airqo-platform/AirQo-frontend
# [MOBILE] implement android widget - v1

#### Summary of Changes (What does this PR do?)

- Create homescreen widget for android application

#### Status of maturity (all need to be checked before merging):

- [] I've included issue number in the "Closes #ISSUE-NUMBER" part of the "[What are the relevant tickets?](#what-are-the-relevant-tickets)" section to [link the issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

#### How should this be manually tested?

- Get this code branch
- Open the `android` directory in android studio.
- Build the application
- check your phone's widget options and select the AirQo widget

#### What are the relevant tickets?

- Closes #<enter issue number here>
- [Jira_card_number]() (If exists)

#### Screenshots (optional)
